client/components/info-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import classNames from 'classnames';
+
+export default class InfoLink extends Component {
+	static propTypes = {
+		href: PropTypes.string.isRequired
+	}
+
+	render() {
+		return (
+			<span
+				ref="infoLink"
+				className={ classNames(
+					'info-link',
+					this.props.className )
+					}
+				>
+				<a href={ this.props.href }>
```

Comment:
I don't really know about this. What if I use `ExternalLink` then instead of `<a>` ? 
@MichaelArestad what do you think? Shall these links open a new tab ?